source/clear-linux/tutorials/smb/smb.rst

```diff
@@ -1,31 +1,56 @@
 .. _clear-samba-share-to-Windows:
 
-Enabling simple file sharing from Clear Linux to a Windows machine using Samba
-##############################################################################
+Enable simple file sharing with a Windows\* machine using Samba\*
+#################################################################
 
-This guide covers the steps you need to enable simple file sharing
-from |CLOSIA| to Windows. For more advanced sharing, please refer to the
-`Samba guide`_.
+This tutorial describes how to enable simple file sharing from a system
+running |CLOSIA| to a Windows machine using Samba. For more advanced sharing,
+refer to the `Samba guide`_.
 
-#.	Follow the :ref:`install-on-target` guide to install |CL|.
-#.	Add the storage-utils bundle, which includes the Samba binaries, after |CL|
-	is setup. The os-clr-on-clr bundle also includes the Samba binaries.
+Prerequisites
+*************
 
-	.. code-block:: console
+This tutorial assumes you have installed |CL| on your host system. For
+detailed instructions, follow the steps in :ref:`bare-metal-install`.
+
+Before you install any new packages, update |CL| with the following command:
+
+.. code-block:: bash
+
+   sudo swupd update
+
+
+Set up file sharing
+*******************
+
+#.	Log in and get root privileges.
+
+	.. code-block:: bash
+
+		sudo -s
+
+#.	Add the storage-utils bundle, which includes the Samba binaries.
+
+	.. code-block:: bash
+
+		swupd bundle-add storage-utils
 
-		$sudo swupd bundle-add storage-utils
+	.. note::
 
-#.	Create a configuration file at :file:`/etc/samba/smb.conf` and add the following
-	(change as needed). The following configuration has two simple examples.
-        The first example demonstrates sharing a folder to a specific user and
-        the second example demonstrates sharing a folder to anyone. The examples
-	assume that a user account named "clearlinuxuser" already exists.
-	If "valid users" is not specified as shown, then anyone with a user account
-	on this machine and with their Samba password already set can access the files.
-        However, that account is only able to access files and folders that the
-        account has permission to on the system. Use :command:`chown`
-	or :command:`chmod` to change either the owner of the file or the
-	permissions to allow other users to access that file.
+		The os-clr-on-clr bundle also includes the Samba binaries.
```

Comment:
Question: is more detail needed about the `os-clr-on-clr` bundle? Or are we saying "If you installed the `os-clr-on-clr` bundle then you don't need this step"? (Line 40 is a Note in rendered output but it looks odd here.)